textscreen/txt_sdl.c

```diff
@@ -626,6 +629,32 @@ static int MouseHasMoved(void)
     }
 }
 
+static int window_focused;
```

Comment:
So, is this static or extern now?

Reply:
It's static in `txt_sdl.c`, maybe I should rename it to avoid confusion.